Engine/Source/GameObject.cpp

```diff
@@ -348,6 +348,8 @@ void GameObject::AddChild(GameObject* child, const int aboveThisId)
 	}
 
 	child->RecalculateLocalTransform();
+	// Recompute active for hierarchy
```

Comment:
This comment do not add new information so I think it should be removed.

Reply:
Hecho